spec/unit/govuk_index/presenters/details_presenter_spec.rb

```diff
@@ -50,31 +14,13 @@
       "will_continue_on" => "on and on",
     }
 
-    presenter = details_presenter(details, "licence")
+    presenter = details_presenter(details)
 
     expect(presenter.licence_identifier).to eq(details["licence_identifier"])
     expect(presenter.licence_short_description).to eq(details["licence_short_description"])
   end
 
-  it "when_additional_indexable_content_keys_are_specified" do
-    details = {
-      "external_related_links" => [],
-      "introductory_paragraph" => [
-        { "content_type" => "text/govspeak", "content" => "**introductory paragraph**" },
-        { "content_type" => "text/html", "content" => "<strong>introductory paragraph</strong>" }
-      ],
-      "more_information" => "more information",
-      "start_button_text" => "Start now",
-    }
-
-    expect(details_presenter(details, %w(introductory_paragraph more_information)).indexable_content).to eq("introductory paragraph\n\nmore information")
-  end
-
-  def details_presenter(details, indexable_content_keys = %w(body parts))
-    described_class.new(
-      details: details,
-      indexable_content_keys: indexable_content_keys,
-      sanitiser: GovukIndex::IndexableContentSanitiser.new
-    )
+  def details_presenter(details)
+    described_class.new(details: details)
```

Comment:
this could easily be moved to a `subject` definition, or given that it is only used once, inlined.